tools/coco_limit_image_size.py

```diff
@@ -52,6 +81,17 @@ def main():
         width_ratio = image.shape[1] / args.limit_image_size[1]
         if height_ratio > 1 or width_ratio > 1:
             ratio = height_ratio if height_ratio > width_ratio else width_ratio
+
+            for i in images_idx_to_annotations_idx[image_info['id']]:
+                content['annotations'][i]['bbox'] = [x / ratio for x in
+                                                     content['annotations'][i]['bbox']]
+                if content['annotations'][i]['segmentation']:
+                    content['annotations'][i]['segmentation'] = resize_segmenation(
+                        content['annotations'][i]['segmentation'],
+                        ratio,
+                        image.shape[0],
+                        image.shape[1]
+                    )
             new_size = int(floor(image.shape[1] / ratio)), int(floor(image.shape[0] / ratio))
```

Comment:
I guess `floor` is redundant here, but feel free to leave as is.

Reply:
fixed